requirements.in

```diff
@@ -1,5 +1,5 @@
 celery==5.2.7
-Django==3.2.16
+Django==3.2.17
```

Comment:
I think we should upgrade to 3.2.18

Reply:
Updated! I also learned that my whole implementation is wrong because webshop's swagger spec wasn't up-to-date. So I'm going to update this PR quite a bit.